manim/_config/utils.py

```diff
@@ -854,6 +859,12 @@ def verbosity(self, val: str) -> None:
         doc="Verbosity level of ffmpeg (no flag).",
     )
 
+    media_width = property(
+        lambda self: self._d["media_width"],
+        lambda self, val: self._d.__setitem__("media_width", val),
+        doc="Media width in Jupyter notebook",
```

Comment:
```suggestion
        doc="Only for jupyter notebooks. Can specify the width of the output video with e.g. `config.media_width=25`, where vw stands for “viewport width”, which is the viewable screen’s width. 100vw would represent 100% of the viewport’s width.",
```

Reply:
I don't know if this is maybe a bit too long, but I think there should be at least some kind of documentation of how to use manim with jupyter notebooks

Reply:
I think this is too long, but in the right direction. We could be a bit more descriptive, but I shorten it so that the meaning is understood. In #1013 with Click, this should be easier to make pretty. If anything should be given an example, it should probably be in the [sphinx docs](https://docs.manim.community/en/v0.4.0/reference/manim.utils.ipython_magic.html?highlight=jupyter).